furiosa/models/vision/efficientnet_v2_s/__init__.py

```diff
@@ -0,0 +1,101 @@
+from pathlib import Path
+from typing import Any, Dict, List, Sequence, Tuple
+
+from PIL import Image, ImageOps
+import numpy as np
+import numpy.typing as npt
+
+from furiosa.registry.model import Format, Metadata, Publication
+
+from ...types import ImageClassificationModel, PostProcessor, PreProcessor
```

Comment:
```suggestion
from ...types import ImageClassificationModel, Platform, PostProcessor, PreProcessor
```

밑의 리뷰 커멘트에서 추가로 필요한 의존성입니다.